ios/22/_sentry_arm64.crash

```diff
@@ -0,0 +1,35 @@
+OS Version: iOS 10.2.1 (14D27)
+Report Version: 104
+
+Exception Type: EXC_BREAKPOINT
+Crashed Thread: 0
+
+Application Specific Information:
+Exception 6, Code 771328, Subcode 8
+
+Thread 0 name:
+Thread 0 Crashed:
+0   CrashLibiOS                     0x1000bc500         @objc CRLCrashSwift.crash() -> ()
+1   CrashLibiOS                     0x1000bc500         CRLCrashSwift.crash() -> () (CRLCrashSwift.swift:36)
```

Comment:
I just looked into this a bit more in detail to see what xcode does. First of all there is definitely a bug in our implementation which I will fix. Inline frames were expanded the wrong way around.

However I did compare to xcode how it displays inline frames and generally it's not very different to what we do. The main difference is that it mentions the inline expansion in the output:

```
  * frame #0: 0x000000010012052c CrashLibiOS`@objc CRLCrashSwift.crash() -> () [inlined] CrashLibiOS.CRLCrashSwift.crash () -> () at CRLCrashSwift.swift:36 [opt]
    frame #1: 0x000000010012052c CrashLibiOS`@objc CRLCrashSwift.crash() -> () at CRLCrashSwift.swift:0 [opt]
```

We can add an `[inline]` into to the stacktrace output. Would that be acceptable?